client/state/selectors/is-gutenberg-enabled.js

```diff
@@ -32,7 +33,11 @@ export const isGutenbergEnabled = ( state, siteId ) => {
 				'>='
 			)
 		) {
-			return isEnabled( 'gutenberg' ) && ! isVipSite( state, siteId );
+			return (
+				isEnabled( 'gutenberg' ) &&
+				! isVipSite( state, siteId ) &&
+				! isPluginActive( state, siteId, 'classic-editor' )
```

Comment:
we should differentiate here between "not active" and "not loaded yet" which I don't believe that `isPluginActive()` does. could this be behind the loop that @obenland found?